### PR TITLE
Draft of RV32E and RV64E chapter for ratification.

### PR DESCRIPTION
**Pull request by kasanovic**

### CODE REVIEW

**Review by aswaterman — APPROVED** [review]

File: src/rv32e.tex [hunk 1]
```diff
@@ -1,58 +1,47 @@
-\chapter{RV32E Base Integer Instruction Set, Version 1.9}
+\chapter{RV32E and RV64E Base Integer Instruction Sets, Version 1.95}
 \label{rv32e}
 
-This chapter describes a draft proposal for the RV32E base integer
-instruction set, which is a reduced version of RV32I designed for
-embedded systems.  The only change is to reduce the number of integer
-registers to 16.  This chapter only outlines the differences between
-RV32E and RV32I, and so should be read after Chapter~\ref{rv32}.
+This chapter describes a proposal for the RV32E and RV64E base integer
```
[the remark this answers is not in the document]
Comment (aswaterman) [reply src/rv32e.tex:4]: I see your point.  The thing being ratified isn't the English, though, it's the underlying technical matter.  So we can freely fix the wording upon ratification.

[the remark this answers is not in the document]
Comment (pdonahue-ventana) [reply src/rv32e.tex:4]: I agree that the wording can be fixed upon ratification.  The problem is that historically that step hasn't been done.  Like:
- https://github.com/riscv/riscv-isa-manual/blob/master/src/c.tex#L5
- https://github.com/riscv/riscv-state-enable/blob/main/content.adoc#proposal
- https://github.com/riscv/riscv-tee/blob/main/Smepmp/Proposal.adoc#proposal
- probably more

[the remark this answers is not in the document]
Comment (aswaterman) [reply src/rv32e.tex:4]: Let's fix that, then.  To that end, I've fixed the C extension text in 42b53c8198194a158895da4cf6ddef9ef627c083.  Thanks for pointing it out.

[the remark this answers is not in the document]
Comment (pdonahue-ventana) [reply src/rv32e.tex:4]: Thanks.  I hadn't noticed the problem with C until just now.  The other two have bothered me for a while, but I guess not enough to make PRs.

**Pull request merged**